### ISSUE TITLE
Window is slightly undersized

### ISSUE THREAD

**Issue by Eiim** [rex706/SAM#184]
It seems like the window is slightly undersized, which causes these unnecessary scroll bars to appear.

![image](https://github.com/rex706/SAM/assets/24627058/33497d0c-6d25-43a9-aa3c-aaf1416c23e7)

Tested on monitors with 100%, 150%, and 300% display scaling, looks identical on each.

Environment: Windows 10

**Comment by rex706** [comment]
Are you building the application yourself and have you modifier the Height property of the Border for the file Menu to be larger by any chance?

**Comment by Eiim** [comment]
No. It's the exact executable file from [the latest release](https://github.com/rex706/SAM/releases/tag/v1.5.3.2).

**Comment by Eiim** [comment]
It seems like if I slightly undersize the account buttons in the settings to 90px, it works fine.

**Comment by rex706** [comment]
Let me know if this still happens with the new version with all the updated packages. Very weird issue

**Comment by Eiim** [comment]
Nice update! Unfortunately, this seems unchanged, although enabling headerless window now adds enough extra space for the icons to fit.
![image](https://github.com/user-attachments/assets/6d38aac7-3dac-443e-9fe2-fc7dd94a1b84)
